pagopa/io-app
# chore: [IOBP-549] Add new wallet payments history tracking

⚠️ Depends on https://github.com/pagopa/io-app/pull/5503

## Short description
This PR implements payment history tracking within the new wallet's payment flow.
Tracking saves all current active payment data and stores the history of the last 15 payments for later retrieval.

## List of changes proposed in this pull request
- Added `history` feature within the wallet's `payment` feature
- Added payment history data to Zendesk's support request

## How to test
Within the payment flow, using Reactotron or any other tool to see the Redux store content, start a payment flow and check the content of `features/wallet/payment/history`. It should be updated with the payment data.
